tools_docker/Debian_Uwsgi/Dockerfile

```diff
@@ -64,7 +64,7 @@ RUN \
     xvfb \
     zlib1g-dev \
 # Install nodejs
-    && curl -sL https://deb.nodesource.com/setup_8.x | bash - \
+    && curl -sL https://deb.nodesource.com/setup_10.x | bash - \
```

Comment:
buster already comes with node 10. I suggest to drop this line and make sure to install npm if not present instead.
```suggestion
```

Reply:
I have tried this before and not find solution to install npm without installing nodejs 10 (with latest version)

Reply:
cancelling this suggestion since buster will not be used.